components/Chips/src/MDCChipCollectionViewCell.h

```diff
@@ -0,0 +1,28 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@class MDCChipView;
+
+/*
+ A collection view cell containing a single MDCChipView.
+ */
+@interface MDCChipCollectionViewCell : UICollectionViewCell
```

Comment:
If MDCChipCollectionViewCell remains public, document how it contains the chip view, e.g. "The contained MDCChipView is resized to fill the cell." (or whatever is true).

Reply:
It may be beneficial to also make MDCChipCollectionViewCell a final class if there isn't any particularly novel logic contained within.

Reply:
I think there is utility in a public MDCChipCell, but agree we should indicate it is not to be subclassed.

Reply:
Making it final would be nice, but if people subclass MDCChipView I would like them to easily be able to use their subclass in tandem with MDCChipCollectionViewCell. Any suggestions that work well with both? `@property(nonatomic) Class chipClass;`?